### ISSUE TITLE
Section component rendered as div element instead of section

### ISSUE THREAD

**Issue by verheyenkoen** [couds/react-bulma-components#34]
According to the Bulma docs, the section component should be written as:

```
<section class="section">
</section>
```

Your `Section` component renders a `div`element instead:

```
<div class="section">
</div>
```

**Comment by threeid** [comment]
@verheyenkoen , you can add `renderAs='section'` to `Section` component, according to
```
static defaultProps = {
    children: null,
    className: '',
    style: {},
    renderAs: 'div',
    size: null,
  }
```

**Comment by couds** [comment]
Fixed on release v1.4.5 [Release Notes](https://github.com/couds/react-bulma-components/releases/tag/v1.4.5)
